hddm/models/hddm_regression.py

```diff
@@ -103,9 +103,7 @@ class HDDMRegressor(HDDM):
 
     def __init__(self, data, models, group_only_regressors=True, keep_regressor_trace=False, **kwargs):
         """Instantiate a regression model.
-
```

Comment:
Why did you remove these empty lines?

Reply:
Sorry, that was not on purpose. Will fix it.

Reply:
Still needs fixing.